vagrant/provisioning/roles/upgrade-solr/tasks/main.yml

```diff
@@ -94,10 +90,10 @@
         name: arkcase
         state: started
       when: _result.status == 200
-  when: solr_data_formated < space_available_formated
+  when: solr_data_formated * 2 > space_available_formated
```

Comment:
@david-oc-miller You were completely right here, and i was neglecting that because it was working (applied the 1st rule, if it works, do not touch it). And took a deeper dive today on this, and found out that math operations are not interpreted the same on conditionals as in the jinja templating {{ something here }}, ex:  solr_data_formated: "{{ solr_data.stdout | human_to_bytes | int * 2 }}" here we double the integer, and here when: solr_data_formated * 2 > space_available_formated we attach the string solr_data_formated twice and we get solr_data_formatedsolr_data_formated.